Library/Homebrew/cask/lib/hbc/cask.rb

```diff
@@ -27,7 +27,7 @@ def metadata_master_container_path
     end
 
     def metadata_versioned_container_path
-      cask_version = version ? version : :unknown
+      cask_version = version ? version.sanitize : :unknown
```

Comment:
In fact, it seems like https://github.com/Homebrew/brew/pull/1502/commits/a799a3979f907307657ef9baddede51fba7d4d18 is sufficient to fix https://github.com/caskroom/homebrew-cask/issues/26630, and the `sanitize` method isn't necessary. Colons are fine, the problem was how we were using `curl`.